### DISCUSSION TITLE
feat(all): Experiment using Popup API

### DISCUSSION THREAD

**Discussion by tbondwilkinson** [angular/components#25884] [Ideas]
### Feature Description

Experiment using the soon to be builtin popup api in components to display them in the top layer.

https://developer.chrome.com/blog/pop-ups-theyre-making-a-resurgence/
https://open-ui.org/components/popup.research.explainer

### Use Case

This would reduce code and eliminate the need for z-index and other positioning hacks.

**Comment by zarend** [comment]
Hello @tbondwilkinson , I converted this issue to a discussion.

-Zach
